oceanbase-ce/README.md

```diff
@@ -50,78 +43,65 @@ boot success!
 
 ## Connect to an OceanBase instance
 
-The `oceanbase-ce` image contains `obclient` (OceanBase Database Client) and the default connection script `ob-mysql`.
+The `oceanbase-ce` image contains `obclient` (OceanBase Database Client) and the default connection script `ob-mysql`. You may refer to the following commands to connect to OceanBase cluster.
```

Comment:
I prefer to remove the ob-mysql script and use normal connection params of mysql client or obclient. The ob-mysql here introduce more complexity.  We can show `docker exec -it oceanbase-ce obclient -uroot ...` here.

Reply:
Yes, so do I